lib/src/model/hms_ios_pip_controller.dart

```diff
@@ -0,0 +1,187 @@
+  }
+
+  /// [start] is used to start PIP manually.
+  ///
+  /// `Note: [setupPIP] is required to call before calling [startPIP].`
+  ///
+  ///Refer [PIP mode guide here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode)
+  static void start() {
+    if (_isPIPSetupDone) {
+      PlatformService.invokeMethod(PlatformMethod.startPIP);
+    }
+  }
+
+  /// [stop] is used to stop PIP manually.
+  ///
+  /// `Note: [setupPIP] is required to call before calling [stopPIP].`
+  ///
+  ///Refer [PIP mode guide here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode)
+  static void stop() {
+    if (_isPIPSetupDone) {
+      PlatformService.invokeMethod(PlatformMethod.stopPIP);
+    }
+  }
+
+  ///[changeTrack] is used to change the track of PIP window.
+  ///
+  ///**Parameters**:
+  ///
+  ///**track** - [HMSVideoTrack] need to be passed for changing PIP window track.
+  ///
+  ///**aspectRatio** - Ratio for PIP window. For example: [16,9], [9,16] ,[1,1]
+  ///
+  /// `Note: [setupPIP] is required to call before calling [changeTrackPIP].`
+  ///
+  /// Refer [PIP mode guide here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode)
+  static Future<HMSException?> changeTrack(
+      {required HMSVideoTrack track,
+      required List<int> aspectRatio,
+      required String alternativeText,
+      required ScaleType scaleType}) async {
+    if (_isPIPSetupDone && aspectRatio.length == 2) {
+      var result = await PlatformService.invokeMethod(
+          PlatformMethod.changeTrackPIP,
+          arguments: {
+            "track_id": track.trackId,
+            "ratio": aspectRatio,
+            "alternative_text": alternativeText,
+            "scale_type": scaleType.value
+          });
+      if (result != null) {
+        return HMSException.fromMap(result["error"]);
+      }
+    } else if (!_isPIPSetupDone) {
+      return HMSException(
+          message:
+              "[setupPIP] is required to call before calling [changeTrackPIP]",
+          description:
+              "[setupPIP] is required to call before calling [changeTrackPIP]",
+          action: "",
+          isTerminal: false);
+    } else if (aspectRatio.length != 2) {
+      return HMSException(
+          message: "ratio must be passed properly",
+          description:
+              "ratio must be passed properly. For example: [16,9], [9,16] ,[1,1]",
+          action: "",
+          isTerminal: false);
+    }
+    return null;
+  }
+
+  ///[changeTrack] is used to change the track of PIP window.
+  ///
+  ///**Parameters**:
+  ///
+  ///**text** - [text] need to be passed for show text in PIP window.
+  ///
+  ///**aspectRatio** - Ratio for PIP window. For example: [16,9], [9,16] ,[1,1]
+  ///
+  /// `Note: [setupPIP] is required to call before calling [changeTextPIP].`
+  ///
+  /// Refer [PIP mode guide here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode)
+  static Future<HMSException?> changeText(
```

Comment:
Mention in comments that calling changeText will remove the video track & only show the Text label